### PR TITLE
add plugin for seo title

### PR DESCRIPTION
**Pull request by dtothefp**
Reverts https://github.com/optimizely/marketing-website/pull/1018 to account for new logic in https://optimizely.atlassian.net/browse/MKT-1696

#### To Test

run `grunt server` 

check that pages with no `TR_seo_title` have in their tab

- if `TR_visible_title` is present, this will be concated with `- Optimizely`
- if `title` is present (partners pages) the dirname is being used, striped of slashes, capitalized, and concated with `- Optimizely`

### CODE REVIEW

File: grunt/assemble/plugins/seo-title.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+       }).join(' ');
+     } else {
+       return cap(str);
+     }
+   }
+
+  /**
+   * Plugin to add SEO title
+   * - if SEO title exists use it
+   * - if title exists then use the capitalized dirname plus a suffix
```
Comment (stewsmith) [inline grunt/assemble/plugins/seo-title.js]: Should this be `if title doesn't exist...`

Comment (dtothefp) [inline grunt/assemble/plugins/seo-title.js]: @stewsmith look at line 66. It checks it the `altTitle` exists, and if not uses the `defaultTitle`

cases:
- seo_title exists => use it 
- visible_title exists => use it + suffix
- title exists => use the dirname + suffix
- nothing exists => use the default title

**Comment by stewsmith** [comment]
retest this please

**Comment by stewsmith** [comment]
LGTM one tests pass

**Pull request merged**